data/sql/updates/pending_db_world/rev_1675207681546911800.sql

```diff
@@ -0,0 +1,10 @@
+--
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 18881;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(18881,0,0,"Sundered Rumbler shatters into shards.",16,0,100,0,0,0,0,0,"Sundered Rumbler - Emote");
```

Comment:
Missing broadcasttextId

You can find it by searching the broadcast_text table

```sql
SELECT * FROM `broadcast_text` WHERE `MaleText` LIKE "%shatters into shards%";
```

Or

```sql
SELECT * FROM `broadcast_text` WHERE `FemaleText` LIKE "%shatters into shards%";
```

Reply:
![image](https://user-images.githubusercontent.com/47818697/218228807-80475330-8d82-4849-9d7c-cdf77019a7c5.png)

Should be 19115